dashboard/app/views/admin_reports/admin_stats.html.haml

```diff
@@ -25,67 +20,45 @@
           Female Students:
           = @stats['born_accounts']['female_students']
           = surround '(', ')' do
-            = number_with_precision(100.0 * @stats['born_accounts']['female_students'].to_f / (@stats['born_accounts']['female_students'].to_f + @stats['born_accounts']['male_students'].to_f), precision: 2)
+            = number_with_precision(100.0 * @stats['born_accounts']['female_students'].to_f / |
```

Comment:
Don't use the `|` character to split up lines. Wrap on commas or extract code into helper.